models/CallLeg.js

```diff
@@ -42,9 +42,9 @@ schema.statics.createCustomerLeg = function (conferenceSid, callSid) {
     });
 };
 
-schema.statics.findCustomer = function (conferenceSid) {
+schema.statics.findCustomerLeg = function (conferenceSid) {
     return new Promise((resolve, reject) => {
-        this.find({ taskSid: conferenceSid, role: 'customer' })
+        this.find({ conferenceSid, role: 'customer' })
```

Comment:
I gotta fix this, mis/un-documented confusion on my part. Turns out a conference has a `FriendlyName` and a sid, as per the docs you can do most operations with just the friendly name. but to put someone on hold you have to get the real sid. I'll go thru and change it everywhere to reflect the reality.

Does `confName` or `confFriendlyName` work?

Reply:
To answer the original question `taskSid` is to be used for the friendlyName if the task is an ACD call. If twilio sees a conference with a taskSid, starts with `WT` it can automate a lot of the work around connecting the call. Unfortunately non of it applies to click to call, you actually need a customer on the line to take advantage of it.

Reply:
I think `confName` is adequate.